### ISSUE TITLE
Edit Account Name and Code in Chart of Accounts after posted entry

### ISSUE THREAD

**Issue by ahetland** [odoo/odoo#24537]
Impacted versions: v11

Steps to reproduce: Go to Accounting (Invoicing) > Accounting > Chart of Accounts select an account where there are posted entries. Click edit to edit the account, change the account name and account code. Click Save.

Current behavior: When saving the edit of an account with posted entries the changes get persisted. 

Expected behavior: It should not be possible to edit account names nor code after entries have been posted to the books.

Video/Screenshot link (optional):

**Comment by Yenthe666** [comment]
@qdp-odoo could you have a look please?

**Comment by qdp-odoo** [comment]
for the record: v8 had that ( https://github.com/odoo/odoo/commit/e242773ca ) but it was removed since v9

**Comment by ahetland** [comment]
This is a legal requirement, you cannot change accounting information after it has been posted. The chart of accounts is the main description of an accounting entry, changing the meaning of entries after they have been posted is not legal and these would change since the link is to the chart of accounts and the original meaning of the account is no longer persisted to the database.

**Comment by pedrobaeza** [comment]
I don't agree with that, @ahetland. Changing the name is not changing accounting information. There are thousand of reasons you can change the name of the account: typo, be more precise...

Any way, this is reversible, so there's no possible harm. As said, don't allow wrong people to change this through permissions.

**Comment by pedrobaeza** [comment]
Reading that extract, I still don't interpret what you are interpreting. Any way, these are interpretations for both. While we don't have what is called in my country "binding query" saying about that, a constraint shouldn't be put.
